### DISCUSSION TITLE
Group

### DISCUSSION THREAD

**Discussion by Linhholi** [ecederstrand/exchangelib#1318] [Q&A] [answered]
Hi, can I access to the emails in a group I am in? (A group not a shared email) If it’s possible, how could I do it? Thank you.

**Comment by ecederstrand** [comment]
Which kind of group are you referring to?

**Reply by Linhholi** [reply]
Hi Erik @ecederstrand,
This is the group that I want to access. It has its own email address.
<img width="276" alt="Screenshot 2024-06-24 at 18 35 06" src="https://github.com/ecederstrand/exchangelib/assets/103567688/3b867c75-2dfb-4457-bdb7-f7f0ca963422">

**Reply by ecederstrand** [reply]
It's still not clear to me what you are referring to. Are you trying to access a mailbox of an account other than your own account? If so, then just create an Account withe the email address of that account: https://ecederstrand.github.io/exchangelib/#setup-and-connecting

Otherwise, please point to some documentation describing the "group" concept in Exchange that you are referring to.

**Reply by Linhholi** [reply]
Hi Erik, this is what I meant:
https://support.microsoft.com/en-au/office/join-a-group-in-outlook-2e59e19c-b872-44c8-ae84-0acc4b79c45d#:~:text=All%20the%20groups%20that%20you,%2C%20notebook%2C%20and%20other%20information.

It’s not a user email so I cannot log in to it like a normal email.

**Reply by ecederstrand** [reply, marked as answer]
I think that's what is called a distribution group in Exchange lingo: https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/distribution-groups-and-ews-in-exchange Those have an email address as well. If you look at one of the emails in one of your groups, you should be able to see the email address of the group. I believe writing to that email address is how the group messages end up under your group in Outlook.

**Reply by Linhholi** [reply]
I believe they don’t have passwords so it is not possible to use credentials to access to it. 

from exchangelib import Credentials

credentials = Credentials(username="MYWINDOMAIN\\myuser", password="topsecret")

**Reply by ecederstrand** [reply]
You don't need the credentials of the distribution list. You use your own credentials to access the mailbox, just like Outlook does. The docs I linked to shows how to do that.

**Reply by Linhholi** [reply]
Everything is sorted out. Thank you Erik.
Great library btw 😀
